docusaurus.config.with-vnext.js

```diff
@@ -0,0 +1,17 @@
+// This file imports the config from docusaurus.config.js and adds the 'current' version to
+// onlyIncludeVersions, so that the 'next' version is available when running
+// 'make start-with-vnext' locally.
+
+let config = require('./docusaurus.config.js');
```

Comment:
We're in the process of moving to Docusuarus v3. Among other things, this drops support for CommonJS. Can you rewrite this to be compatible with ESM?

Reply:
@ctauchen Apologies for the extremely long delay. Let me begin by admitting js is as far as it can get from my strong suite, so let me know if I'm missing something. I tried updating into using `import / export` instead of `require()`, but kept getting errors, like this one:
```
[ERROR] Error [ERR_REQUIRE_ESM]: require() of ES Module docusaurus.config.with-vnext.mjs not supported.
```
(This was after I changed the extension to `.mjs` to indicate that this was a module, otherwise I'd get a 'cannot import outside a module' error.)

I think this is because of the original `docusaurus.config.js` file that this one imports, so it's like a chicken and egg problem. I did add a 'TODO' comment with the new syntax (as far as I can tell), but I don't think I can update this before the docusaurus v3 upgrade...

Reply:
Thanks, I've had a look and tested and all looks good. We'll adjust the script for the Docusuarus 3 upgrade when we get there.